File: OpenMPvalidation/SRC/F90/omp_small_copyprivate.f90
	SUBROUTINE init()
	real:: x = 0.0
!$omp threadprivate(x)

!$omp single 
	x = 1.0
!$omp end single copyprivate(x)
	end

	program main
!$omp parallel
	call init()
!$omp end parallel
	end program main

File: OpenMPvalidation/SRC/C/omp_small_copyprivate.c
#include <omp.h>

float x=0.0;
int y=0;
#pragma omp threadprivate(x,y)

void init ()
{
  #pragma omp single copyprivate(x,y)
  {
    x=1.0;
    y=1;
  }
}

int main (int argc, char * argv[])
{
#pragma omp parallel
  {
    init ();
  }
  return 0;
}
